src/renderer/components/settings/Preferences.vue

```diff
@@ -64,6 +64,7 @@ export default {
       this.$services.network.setSelectedNetwork(network);
       this.$store.commit('handleNetworkChange');
       this.$store.dispatch('fetchPortfolio');
+      this.$store.dispatch('fetchLatestVersion');
```

Comment:
Nice improvement; I noticed if they started on MainNet and switched to TestNet that the dex wouldn't correctly detect out of date. This seems like a reasonable fix for it.